packages/chakra-components/docs/03-Election.mdx

```diff
@@ -36,21 +36,33 @@ Show election title.
 
 ## Election Schedule
 
-Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format.
+Show election period. You can specify date format using [date-fns](https://date-fns.org/v2.30.0/docs/format) format
 
 ```jsx live
 <ElectionSchedule format={'d-L-y HH:mm'} />
 ```
 
 ## Election Status Badge
 
-It shows election status badge. Possible values of the election are defined on `ElectionStatus` from `@vocdoni/sdk`:
+It shows election status badge. Possible values of the election are defined on `ElectionStatus` from [`@vocdoni/sdk`](https://github.com/vocdoni/vocdoni-sdk/blob/main/src/types/election/published.ts#L6-L18):
 `PROCESS_UNKNOWN`, `UPCOMING`, `ONGOING`, `ENDED`, `CANCELED`, `PAUSED`, `RESULTS`.
 
 ```jsx live
 <ElectionStatusBadge variant='solid' colorScheme='teal' />
 ```
 
+To style the [`ElectionStatusBadge`](https://chakra-ui.com/docs/components/tag) leverage the capabilities of the Chakra
+styled system. By simply adjusting the `colorScheme` property based on the election status will enable you to easily alter the
+color of the badge. This approach mirrors the one employed by the
+[default color scheme](https://github.com/vocdoni/ui-components/blob/main/packages/chakra-components/src/components/Election/StatusBadge.tsx#L12-L24)
+included on the Vocdoni Chakra Components.
+
+For a more comprehensive customization, consider creating a custom variant as demonstrated in the
+[chakra docs](https://chakra-ui.com/docs/components/tag/theming#adding-a-custom-variant).
```

Comment:
I don't agree with this sentence. It's even easier to replace the base styles for the tag component, rather than trying to understand how variants work. Also, variants and base styles are basic styling features of chakra. I think these sentences would be better in a more generic way under the "Theming" section.

Reply:
The purpose of this sentence is to easily point the end user to the right direction in case deeper customization is needed. After reading chakra components `README.md`  the easier wy is to point the user to theming section, where this is explained

Reply:
Force pushed fix bfc1aa7